apps/web/src/app/(dashboard)/(management)/penningmeester/_components/queries.ts

```
@@ -0,0 +1,171 @@
+ * Aggregated in SQL (GROUP BY) and uses `count(distinct certificate.id)` so the
+ * join path can never inflate a count. Only locations with >= 1 counted
+ * certificate appear, regardless of location.status (an archived/hidden
+ * location with billable activity must still be invoiced).
+ */
+export async function listCertificateCountsByLocation(
+  from: string,
+  to: string,
+): Promise<LocationCertificateCounts[]> {
+  const user = await assertCanViewFinancialReport();
+  const { fromUtc, toUtcExclusive } = amsterdamPeriodToUtcBounds(from, to);
+  const query = useQuery();
```

Biome (2.4.15)

[error] This hook is being called from within a function or method that is not a hook or component.

(lint/correctness/useHookAtTopLevel)